CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Fixed an issue where author image validations were not checked properly.
 * Fixed an issue where new old-formatted scripts and integrations were not validated.
 * Fixed an issue where the wording in the from version validation error for subplaybooks was incorrect.
+* Fixed an issue where **update-release-notes** command used the old docker image version instead of the new when the docker was changed.
```

Comment:
```suggestion
* Fixed an issue where the **update-release-notes** command used the old docker image version instead of the new when detecting a docker change.
```